nodemcu/lua/led.lua

```diff
@@ -16,8 +19,13 @@ local function start()
           gpio.write(config.ledRed, gpio.HIGH)
         end
         if mode == 1 then
-          gpio.write(config.ledBlue, gpio.LOW)
-          gpio.write(config.ledRed, gpio.HIGH)
+          if config.ledState == 1 then
+            gpio.write(config.ledBlue, gpio.LOW)
+            gpio.write(config.ledRed, gpio.HIGH)
+          else
```

Comment:
I have to agree it's not very explicit but this else case is basically the option 2 case